src/main/java/net/springjfx/AbstractFxmlView.java

```diff
@@ -1,4 +1,4 @@
-package de.felixroske.jfxsupport;
+package net.springjfx;
```

Comment:
This will make all existing projects backwards incompatible when upgrading to a newer version. To rename packages in a smooth manner consider subclassing the existing classes under the new package name and mark this one as @Deprecated and subject to be removed until Jan 2018 (or whatever).